jsx/StaticDataTable.js

```diff
@@ -66,6 +67,10 @@ var StaticDataTable = React.createClass({
         $("#dynamictable").DynamicTable();
       }
     }
+    if (this.props.onSort) {
```

Comment:
I don't feel like `componentDidMount` is the right place to put this, but I don't really have any better suggestions since `setSortColumn` is called before it's sorted, not after.

Maybe you can add an internal state variable in the setSortColumn closure and check/unset it here, to prevent the callback from firing on non-sort related updates?